com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1334,34 +1356,48 @@ private void ConfigureSimulator()
                 maxConnectAttempts: transport.m_MaxConnectAttempts,
                 connectTimeoutMS: transport.m_ConnectTimeoutMS,
                 disconnectTimeoutMS: transport.m_DisconnectTimeoutMS,
+#if UTP_TRANSPORT_2_0_ABOVE
+                sendQueueCapacity: m_MaxPacketQueueSize,
+                receiveQueueCapacity: m_MaxPacketQueueSize,
+#endif
                 heartbeatTimeoutMS: transport.m_HeartbeatTimeoutMS);
 
             driver = NetworkDriver.Create(m_NetworkSettings);
 
+#if UTP_TRANSPORT_2_0_ABOVE
```

Comment:
Doesn't this need to be wrapped in `#if MULTIPLAYER_TOOLS`? Have you tested without tools package after your changes?

Reply:
You are absolutely right, will fix. thanks.